sunpy/io/tests/test_jp2.py

```diff
@@ -1,12 +1,14 @@
 import numpy as np
+import sunpy.io.jp2 as jp2
```

Comment:
Below we do` from sunpy.io.jp2 import get_header, read`, these imports should be unified.

Either adding write there or changing all the others to be `jp2.read` etc

Reply:
I'll change others to jp2.get_header and jp2.read

read/write alone are too ambiguous so I added the extra import.